app/models/app.rb

```diff
@@ -23,12 +23,9 @@ class App < ActiveRecord::Base
   @@DEPLOYMENT_STATUSES = [:dead, :development, :in_use, :in_use_and_wants_improvement, :inactive_but_wants_improvement, :pending]
 
   enum comment_type: [:contact_status, :app_functionality, :general, :vetting]
-
-  default_scope { order(:name => :asc) }
-  scope :featured, -> { where.not("status = ? or status = ?", App.statuses[:dead], App.statuses[:pending]) }
-  
   
-   
+  #default_scope { order(:name => :asc) }
```

Comment:
Ideally, this diff should not have changed this file.
Kindly either update the diff or follow up with a diff that reverses this.